x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/crossclusteraccess/CrossClusterAccessHeadersForCcsRestIT.java

```diff
@@ -758,6 +758,72 @@ public void testRemoteClustersXPackUsage() throws IOException {
         }
     }
 
+    public void testCrossClusterApiKeyUsage() throws IOException {
+        // Randomly create REST API keys
+        for (int i = 0; i < randomIntBetween(0, 2); i++) {
+            createOrGrantApiKey(Strings.format("""
+                {
+                  "name": "my-rest-key-%s"
+                }
+                """, i));
+        }
+
+        // Randomly create cross-cluster API keys
+        final int ccsKeys = randomIntBetween(0, 2);
+        for (int i = 0; i < ccsKeys; i++) {
+            final Request request = new Request("POST", "/_security/cross_cluster/api_key");
+            request.setJsonEntity("""
+                {
+                  "name": "my-cross-cluster-key-%s",
```

Comment:
I added `String.format` and also changed the base names to be different for different cross-cluster API keys, e.g. `my-ccs-key-xx`, `my-ccr-key-xx`. Like you said, the names do not really matter. But my original intention was to have them to be distinct just in case we need to debug in future.